frontend/src/routes/playground/+page.svelte

```diff
@@ -42,9 +43,17 @@
 </div>
 
 <div class="flex h-full w-full flex-col p-4">
-	<h1 class="mb-2 text-2xl font-bold">Zeno View Playground</h1>
+	<div class="mb-4 flex h-12 items-center">
+		<a href="/" class="shrink-0">
+			<img src="/zeno-logo.png" class="h-8" alt="diamond tesselation logo" />
+		</a>
+		<h1 class="ml-3 text-2xl font-bold">Instance View Playground</h1>
```

Comment:
This looks weird now. Maybe move below the header or remove?

Reply:
Hmm looks okay to me? which part is off?